### PR TITLE
Attach env vars from kube config when renewing token

### PR DESCRIPTION
**Pull request by cscetbon**
Solves #207

### CODE REVIEW

File: lib/helpers.js [hunk 1]
```diff
@@ -132,10 +132,13 @@ function getToken(userInfo) {
       if (profile) {
         envvars.AWS_PROFILE = profile.value;
       }
+      for (const envvar of env) {
+        envvars[envvar.name] = envvar.value ?? ""
```
Comment (andresmgot) [inline lib/helpers.js:136]: why not `envvar.value || ""`?

Also, the `if(profile)` above is not needed anymore, right?

**Comment by cscetbon** [comment]
> Thanks for the PR @cscetbon ! I have a minor comment. Also, can you bump the package version so this will be automatically released once merged?

@andresmgot done, see my last commits. It seems there are issues in your travis scripts

**Comment by andresmgot** [comment]
Thanks for the changes! A bit difficult to spot, but your changes have some lint errors:

```
/home/travis/build/serverless/serverless-kubeless/lib/helpers.js
  132:48  error  Strings must use singlequote    quotes
  132:50  error  Missing semicolon               semi
  137:35  error  A space is required after '{'   object-curly-spacing
  137:48  error  A space is required before '}'  object-curly-spacing
```

Once that's fixed we can land this

**Comment by cscetbon** [comment]
@andresmgot I fixed those errors. Not sure what Travis complains about.

**Comment by andresmgot** [comment]
Seems just a transient error with xip.io. This is fine now, thanks!

**Pull request merged**